lib/react.dart

```diff
@@ -20,6 +20,7 @@ export 'package:react/react_client/react_interop.dart' show forwardRef, createRe
 typedef Error PropValidator<TProps>(
     TProps props, String propName, String componentName, String location, String propFullName);
 
+typedef FunctionComponent = dynamic Function(Map props);
```

Comment:
Should the return type be `ReactElement`? Or do function components support fragments as well?

Reply:
its typed as dynamic because dart doesn't support union types or inheriting primitive types, React16 allows for a lot of things be returned... string, list, numbers, fragment, react element, etc...